daffodil-core/src/test/scala/org/apache/daffodil/core/dpath/TestDFDLExpressionEvaluation.scala

```diff
@@ -66,7 +67,7 @@ class TestDFDLExpressionEvaluation extends Parsers {
     val doc = infosetRootElem.parent.asInstanceOf[InfosetDocument]
 
     // Below, dis is fake. Zero bits available.
-    using(InputSourceDataInputStream(ByteBuffer.allocate(0))) { dis =>
+    Using(InputSourceDataInputStream(ByteBuffer.allocate(0))) { dis =>
```

Comment:
Do we want to use `Using.resource` here? `Using.apply` returns a `Try` instead of throwing exceptions, so any thrown exceptions will be silently ignored, which could mean a test would pass when it shouldn't.